src/Hng.Domain/Entities/Models/User.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+using Microsoft.AspNetCore.Identity;
+
+namespace Hng.Domain.Entities.Models
+{
+    public class User : IdentityUser<long>
```

Comment:
Don't do this. Don't use ASP.net identity service. Just create a plain user table. There are tickets for creating custom roles and claims and all that stuff. We don't need to hook up to ASP.net specific implementation.